make_plant_parts_eia.py

```diff
@@ -869,26 +939,45 @@ def dedup_on_category(self, dedup_df, base_cols, category_name, sorter):
         dedup_df = dedup_df.sort_values(category_name)
         return dedup_df.drop_duplicates(subset=base_cols, keep='first')
 
-    def grab_qualifiers(self,
-                        part_df,
-                        record_name,
-                        id_cols,
-                        denorm_table,
-                        denorm_cols
-                        ):
+    def get_qualifiers(self,
+                       part_df,
+                       record_name,
+                       id_cols,
+                       denorm_table,
+                       denorm_cols
+                       ):
         """
-        Grab qualifier records.
-
-        For an individual compiled dataframe for each of the plant parts, we
-        want to add back in some non-data columns (qualifier columns). When
-        qualitative data is consistent for every record in a plant part, we
-        assign these catagoricals. If the records are not consistent, then
-        nothing is added.
+        Get qualifier records.
+
+        For an individual dataframe of one plant part, we typically have
+        identifying columns and aggregated data columns. The identifying
+        columns are only the identifying columns associated with that plant
+        part (i.e. for plant_unit (plant part) we have ('plant_id_eia' and
+        'unit_id_pudl')). Every plant part is cobbled together from generator
```

Comment:
The identifying columns for a given plant part are only those columns which are required to uniquely specify a record of that type of plant part. For example, to uniquely specify a `plant_unit` record, we need both `plant_id_eia` and the `unit_id_pudl`, and nothing else. In other words, the identifying columns for a given plant part would make up a natural composite primary key for a table composed entirely of that type of plant part.